### PR TITLE
Target net45

### PR DESCRIPTION
**Pull request by bricelam**
@anpete would like to use this in EF

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@bricelam__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

File: src/Microsoft.Framework.Cache.Memory/IMemoryCache.cs [hunk 1]
```diff
@@ -2,11 +2,15 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+#if !NET45
 using Microsoft.Framework.Runtime;
+#endif
 
 namespace Microsoft.Framework.Cache.Memory
 {
+#if !NET45
```
Comment (Praburaj) [inline src/Microsoft.Framework.Cache.Memory/IMemoryCache.cs]: Can we change these to `#if ASPNET50 || ASPNETCORE50` instead. If in future we extend this more frameworks we don't have to change this.

**Comment by Eilon** [comment]
Looks :shipit: to me.

**Comment by Praburaj** [comment]
:shipit:

**Pull request merged**